hack/quickstart-scc.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -e
+
+# parameters from caller deploy-quickstart.sh
+PROVIDER="${1:-ollama}"
+NAMESPACE="${PROVIDER}-dist"
+SERVICE_ACCOUNT="${PROVIDER}-sa"
+SCC_NAME="${PROVIDER}-scc"
+
+echo "Checking if OpenShift prerequisites exist for namespace: ${NAMESPACE}, service account: ${SERVICE_ACCOUNT}..."
+if ! kubectl get scc ${SCC_NAME} &> /dev/null || \
+   ! kubectl get sa ${SERVICE_ACCOUNT} -n ${NAMESPACE} &> /dev/null || \
+   ! kubectl get role ${SCC_NAME}-role -n ${NAMESPACE} &> /dev/null || \
+   ! kubectl get rolebinding ${SCC_NAME}-rolebinding -n ${NAMESPACE} &> /dev/null; then
```

Comment:
If you wanted, you could wrap these statements in functions to turn the check itself into a 1-liner, something like:

```
scc_exists() { kubectl get scc "${SCC_NAME}" &> /dev/null; }
sa_exists() { kubectl get sa "${SERVICE_ACCOUNT}" -n "${NAMESPACE}" &> /dev/null; }
role_exists() { kubectl get role "${SCC_NAME}-role" -n "${NAMESPACE}" &> /dev/null; }
rolebinding_exists() { kubectl get rolebinding "${SCC_NAME}-rolebinding" -n "${NAMESPACE}" &> /dev/null; }

if ! scc_exists || ! sa_exists || ! role_exists || ! rolebinding_exists; then
```

It's already readable though, so up to you.